### ISSUE TITLE
Lab Report2 Feedback

### ISSUE THREAD

**Issue by MohaElder** [CNsquared/cse15l-lab-reports#2]
**Bug/Input 1**
- [x] Commit screenshot for bug/input 1
- [x] Link to test file for bug/input 1
- [x] Symptom shown as command + output for bug/input 1
- [x] Description of relationship between bug, symptom, and input for bug/input 1

**Bug/Input 2**
- [x] Commit screenshot for bug/input 2
- [x] Link to test file for bug/input 2
- [x] Symptom shown as command + output for bug/input 2
- [x] Description of relationship between bug, symptom, and input for bug/input 2

**Bug/Input 3**
- [x] Commit screenshot for bug/input 3
- [x] Link to test file for bug/input 3
- [x] Symptom shown as command + output for bug/input 3
- [x] Description of relationship between bug, symptom, and input for bug/input 3

Additional comments from your grader:
Good job!

Feel free to comment below if you have any questions about the rubric/feedback.

--------

If you don't have all the checkmarks above, fix the issues described, and then submit a [regrade request on Gradescope](https://help.gradescope.com/article/8hchz9h8wh-student-regrade-request). Use this template for your regrade request:

- Link to this issue
- Link to your published lab report page

**Comment by CNsquared** [comment]
Wait does this mean that I don't have to submit regrade request?

**Comment by MohaElder** [comment]
Yup you’re good, you should be seeing your grade soon this or next week

Bests,
Yasushi Oh
________________________________
发件人: CNsquared ***@***.***>
发送时间: Thursday, February 3, 2022 9:55:06 PM
收件人: CNsquared/cse15l-lab-reports ***@***.***>
抄送: MohaElder169 ***@***.***>; Author ***@***.***>
主题: Re: [CNsquared/cse15l-lab-reports] Lab Report2 Feedback (Issue #2)


Wait does this mean that I don't have to submit regrade request?

―
Reply to this email directly, view it on GitHub<https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FCNsquared%2Fcse15l-lab-reports%2Fissues%2F2%23issuecomment-1029682844&data=04%7C01%7C%7Cebe62a8120b9460a637c08d9e7a2e70c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637795509122223959%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=gMuFa1X4hkq9%2FtSD3R7vy8qAhI3j5eh4mn7EsWxYfig%3D&reserved=0>, or unsubscribe<https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAE7Z4SHP26CHIZLGRFFEHR3UZNS3VANCNFSM5NQZC5PA&data=04%7C01%7C%7Cebe62a8120b9460a637c08d9e7a2e70c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637795509122223959%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=cuMt%2FlWFQrGVt33HskK%2FLtTSp%2F4Jy3mqhASJXJKP0MY%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7C%7Cebe62a8120b9460a637c08d9e7a2e70c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637795509122223959%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=Wr2UFwHdwKX2bX1n42rIZe9m9O2iJfohKoLygy4THdc%3D&reserved=0> or Android<https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7C%7Cebe62a8120b9460a637c08d9e7a2e70c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637795509122223959%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=Q5EtzKsQxQddd10BHzeJQAiCp3jVmKMMdVjAJzxFtrY%3D&reserved=0>.
